OptimizelySDK/Config/DatafileProjectConfig.cs

```diff
@@ -636,6 +636,11 @@ public Variation GetFlagVariationByKey(string flagKey, string variationKey)
         /// <returns>Rollout Entity corresponding to the rollout ID or a dummy entity if ID is invalid</returns>
         public Rollout GetRolloutFromId(string rolloutId)
         {
+            if (string.IsNullOrEmpty(rolloutId.Trim()))
```

Comment:
this will cause exception in case rolloutId is missing and eventually `null` is passed. Check this method 
`string.IsNullOrWhiteSpace`

Reply:
@msohailhussain That's an excellent point.  I normally use `string.IsNullOrWhiteSpace` but it was failing build for NET 3.5.  

Should I use preprocessor directives? `#if NET35 ... #elif ... #endif`?